jans-cedarling/cedarling/src/authz/entities/workload.rs

```diff
@@ -0,0 +1,261 @@
+/*
+ * This software is available under the Apache-2.0 license.
+ * See https://www.apache.org/licenses/LICENSE-2.0.txt for full text.
+ *
+ * Copyright (c) 2024, Gluu, Inc.
+ */
+
+use super::{CreateCedarEntityError, DecodedTokens, EntityMetadata, EntityParsedTypeName};
+use crate::common::policy_store::{PolicyStore, TokenKind};
+use std::collections::HashSet;
+use std::fmt;
+
+/// Create workload entity
+pub fn create_workload_entity(
+    entity_mapping: Option<&str>,
+    policy_store: &PolicyStore,
+    tokens: &DecodedTokens,
+) -> Result<cedar_policy::Entity, CreateWorkloadEntityError> {
+    let namespace = policy_store.namespace();
+    let schema = &policy_store.schema.json;
+    let mut errors = Vec::new();
+
+    if let Some(token) = tokens.id_token.as_ref() {
```

Comment:
Please, describe why you decided that we need try to create user entity from each token?

Reply:
there has been a discussion about this. Cedarling should create the entities to the best of its ability based on the token(s) it has.